shared/constants/swaps.js

```diff
@@ -9,17 +18,42 @@ export const ETH_SWAPS_TOKEN_OBJECT = {
   iconUrl: 'images/black-eth-logo.svg',
 };
 
-export const QUOTES_EXPIRED_ERROR = 'quotes-expired';
-export const SWAP_FAILED_ERROR = 'swap-failed-error';
-export const ERROR_FETCHING_QUOTES = 'error-fetching-quotes';
-export const QUOTES_NOT_AVAILABLE_ERROR = 'quotes-not-avilable';
-export const OFFLINE_FOR_MAINTENANCE = 'offline-for-maintenance';
-export const SWAPS_FETCH_ORDER_CONFLICT = 'swaps-fetch-order-conflict';
+const TEST_ETH_SWAPS_TOKEN_OBJECT = {
+  symbol: 'TESTETH',
+  name: 'Test Ether',
+  address: ETH_SWAPS_TOKEN_ADDRESS,
+  decimals: 18,
+  iconUrl: 'images/black-eth-logo.svg',
+};
 
 // A gas value for ERC20 approve calls that should be sufficient for all ERC20 approve implementations
 export const DEFAULT_ERC20_APPROVE_GAS = '0x1d4c0';
 
-export const SWAPS_CONTRACT_ADDRESS =
-  '0x881d40237659c251811cec9c364ef91dc08d300c';
+const MAINNET_CONTRACT_ADDRESS = '0x881d40237659c251811cec9c364ef91dc08d300c';
+
+const TESTNET_CONTRACT_ADDRESS = '0x881d40237659c251811cec9c364ef91dc08d300c';
+
+const METASWAP_ETH_API_HOST = 'https://api.metaswap.codefi.network';
+
+const SWAPS_TESTNET_CHAIN_ID = '0x539';
```

Comment:
what prevents a local ganache that doesn't fork mainnet or have the ability to interact with swaps from using this API?

Reply:
hmmm.... good point... nothing prevents that

Reply:
I think that's okay though... no risk of fund loss, and it is only developers using local ganache